overrides/config/betterquesting/DefaultQuests.json

```diff
@@ -97639,7 +97645,7 @@
           "repeat_relative:1": 1,
           "ismain:1": 0,
           "name:8": "No Pressure though",
-          "desc:8": "Craft a Mekanism Pressurized Reaction Chamber! It takes 1 Fluid input, 1 Gas input and 1 Item input to create other Gases and Items. You\u0027ll also need to provide it with power, and the back side is the only one that can accept Liquids, so use all sides wisely!"
+          "desc:8": "Craft a Mekanism Pressurized Reaction Chamber! It takes 1 Fluid input, 1 Gas input and 1 Item input to create other Gases and Items. You\u0027ll also need to provide it with power, and the back side is the only one that can accept Liquids by default, so use all sides wisely!"
```

Comment:
this seems weird and disjointed - the "the back side can only accept liquids by default" should be cut - how does `You\u0027ll also need to provide it with power, so make sure to use all sides wisely!`